listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -252,6 +275,7 @@ def process_all_spotify_users():
 def main():
     app = listenbrainz.webserver.create_app()
     with app.app_context():
+        musicbrainz_db.init_db_engine(current_app.config['MUSICBRAINZ_DB_URI'])
```

Comment:
Why is this not done in `create_app()`? (https://github.com/metabrainz/listenbrainz-server/blob/72694d6ef073bfa2a4c0506e2ca8ee39348c5a7a/listenbrainz/webserver/__init__.py#L116)
Why does this script specifically need musicbrainz access? (and not the rest of LB?)

Reply:
Moved to create_app. I had some flawed thinking for keeping it in spotify_read_listens, right now this is the only script that requires access to the MB db to get user emails, so I was trying to keep it here, but we'll have to eventually move it to create_app anyways, so that is the best place.